laboratory-maniac.md

```diff
@@ -84,8 +109,6 @@ singleton Chromatic Sphere is not a huge price to pay, because it can
 generally also be used to draw into your Doomsday pile or just cantrip 
 while fixing mana colors.
 
-To use Chromatic Sphere inside Doomsday piles that do not use SDT to 
-draw into the pile simply replace the SDT in the pile with a Chromatic 
-Sphere and use one extra mana to draw from an empty library. Piles that 
-use Sensei's Divining Top to draw into the pile can't use Chromatic 
-Sphere very effectively.
+To use Chromatic Sphere inside Doomsday piles that do not use CB to 
```

Comment:
With SDT it mattered whether or not you'd draw into the pile with it or with a cantrip. With CB this is not the case. If you have a single-cantrip IU pile that uses LM you can replace the CB in the pile with a Chromatic Sphere. For double-cantrip IU piles the Chromatic Sphere comes in place of the GP, and increases mana cost with 2 rather than 1.